docs/docs/User Guide/Operations/Chunks to NAR Migration.md

```
@@ -158,7 +158,7 @@ When OpenTelemetry is enabled (`--otel-enabled`), the process exports:
 
 ## Verification
 
-```mariadb
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)